klpt/data/kmr-Latn.dic

```diff
@@ -21246,4 +21248,654 @@ zûxal/F po:noun_fem
 şûşekirî/AN po:adj
 şûşt/T po:verb is:past_stem_transitive_active st:şûşt lem:şûştin
 şûştin/F po:noun_fem
-şûştin/XF po:verb is:infinitive_transitive_active st:şûşt lem:şûştin
\ No newline at end of file
+şûştin/XF po:verb is:infinitive_transitive_active st:şûşt lem:şûştin
+
+alîkarî kirin
+jîndar
+pûç
+hindik
+dijwar
+avis
+girîng
+bifêde
+bi kar anîn
+fêr bûn
+şimik
+hez kirin
+peya kirin
+wisa
+xirab
+çavî
+fermo
+lê belê
+tew
+seh kirin
+ketin destê (yekî)
+hêrs bûn
+reben
+qerf kirin
+dest bi (tiştekî) kirin
+zer
+guhdar kirin
+rojava
+weke
+rizgar kirin
+ji bîr kirin
+çekdar
+li pey
+li dû (yekî) kişandin
+li dû
+xasî
+sivik
+dirêj
+neçar
+hewş
+qifil
+nizm
+kûr
+tirş
+veciniqandin
+xilas
+mêr
+lorandin
+hêja
+giran
+bira
+fediyok
+bêhempa
+hinek
+çêtir
+nalîn
+pak
+qedexekirî
+hewl dan
+lîçik
+nenûk
+mezintir
+bilind
+carek
+geş
+beşdar bûn
+serfiraz
+aşkere kirin
+kelacanî
+zindî
+erzanî
+pîroz
+dewlemend
+hîn bûn
```

Comment:
I will add morfosyntactic tag. I need to know how you write multi-word expressions